server/controllers/finance/reports/cashflow/report.handlebars

```diff
@@ -62,14 +62,17 @@
                 <td colspan="{{colspan}}">&nbsp;</td>
               </tr>
             {{/each}}
+            <tr>
+              <td colspan="{{colspan}}">&nbsp;</td>
+            </tr>
           </tbody>
 
           <tbody>
             {{!-- EXPENSES --}}
             <tr style="background-color: #efefef;font-size:1.3rem;">
               <th class="text-left text-uppercase">{{translate 'REPORT.CASHFLOW.EXPENSES'}}</th>
               {{#each periods as |period| }}
-                <th class="text-right">{{debcred (look ../expenseTotal period) ../metadata.enterprise.currency_id}}</th>
+                <th class="text-right">{{debcred (multiply (look ../expenseTotal period) -1) ../metadata.enterprise.currency_id}}</th>
```

Comment:
For a fund to spend money the account is credited, and to know the balance of this account it is done by debit - credit, so that the balance will be negative, and for posting the report Cashflow values must be positive, which is why I have multiplied the values of transactions with the standard transaction affected by expence by (-1)